traffic_ops/app/db/migrations/20161216094845_add_scope_parameters.sql

```diff
@@ -0,0 +1,146 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+INSERT INTO
+   PARAMETER (name, config_file, VALUE) 
+   SELECT
+      'scope',
+      config_file,
+      'server' 
+   FROM
+      PARAMETER 
+   WHERE
+      name = 'location' 
+      AND config_file LIKE 'to_ext_%' 
+      OR name = 'location' 
+      AND config_file = '12M_facts' 
+      OR name = 'location' 
+      AND config_file = 'ip_allow.config' 
+      OR name = 'location' 
+      AND config_file = 'parent.config' 
+      OR name = 'location' 
+      AND config_file = 'records.config' 
+      OR name = 'location' 
+      AND config_file = 'remap.config' 
+      OR name = 'location' 
+      AND config_file = 'hosting.config' 
+      OR name = 'location' 
+      AND config_file = 'traffic_ops_ort_syncds.cron' 
+      OR name = 'location' 
+      AND config_file = 'cache.config';
+INSERT INTO
+   PARAMETER (name, config_file, VALUE) 
+   SELECT
+      'scope',
+      config_file,
+      'cdn' 
+   FROM
+      PARAMETER 
+   WHERE
+      name = 'location' 
+      AND config_file LIKE 'cacheurl%' 
+      OR name = 'location' 
+      AND config_file LIKE 'hdr_rw_%' 
+      OR name = 'location' 
+      AND config_file LIKE 'regex_remap_%' 
+      OR name = 'location' 
+      AND config_file = 'regex_revalidate.config' 
+      OR name = 'location' 
+      AND config_file LIKE 'set_dscp_%' 
+      OR name = 'location' 
+      AND config_file = 'ssl_multicert.config' 
+      OR name = 'location' 
+      AND config_file = 'bg_fetch.config';
+INSERT INTO
+   PARAMETER (name, config_file, VALUE) 
+   SELECT
+      'scope',
+      config_file,
+      'profile' 
+   FROM
+      PARAMETER 
+   WHERE
+      name = 'location' 
+      AND config_file = '50-ats.rules' 
+      OR name = 'location' 
+      AND config_file = 'astats.config' 
+      OR name = 'location' 
+      AND config_file = 'drop_qstring.config' 
+      OR name = 'location' 
+      AND config_file = 'logs_xml.config' 
+      OR name = 'location' 
+      AND config_file = 'plugin.config' 
+      OR name = 'location' 
+      AND config_file = 'storage.config' 
+      OR name = 'location' 
+      AND config_file = 'sysctl.conf' 
+      OR name = 'location' 
+      AND config_file LIKE 'url_sig_%' 
+      OR name = 'location' 
+      AND config_file = 'volume.config';
+INSERT INTO
```

Comment:
I don't really have any idea what this query below (profile_parameter) is doing. the queries above I understand. you are creating scope parameters for each config file but i can't figure out what the intent of the following profile_parameter insert is. maybe add some comments?